docs/configuration/authentication/digid.rst

```diff
@@ -4,22 +4,59 @@
 DigiD authentication
 ====================
 
-.. warning::
-
-    This plugin cannot be configured via the admin interface and requires an
-    update of the Open Forms installation.
-
 Some forms can require authentication. Open Forms supports authentication
 using `DigiD`_. Access to DigiD can typically be obtained via `Logius`_.
 
 Using DigiD for authentication will provide the BSN (social security number) of
 the authenticated person to the form context. Using the BSN, certain fields can
 be :ref:`prefilled <configuration_prefill_index>` with relevant personal data.
 
-.. note::
+Steps to request access to a DigiD environment
+==============================================
+
+The high-level overview of steps you need to perform are described here. The following
+sections provide more details.
+
+1. Read the requirements for getting access to DigiD on the `Logius`_ website.
+   There are several steps that need to be taken on your end that are not
+   covered here.
+
+2. Request a `PKIoverheid Private Services Server G1`_ certificate at your
+   `PKIO SSL certificate supplier`_. This is required for backchannel
+   communication with Logius (if you already have one for Open Forms, it can be
+   re-used).
+
+   You can :ref:`prepare the certificates <configuration_authentication_digid_prepare_certificates>`
+   from the admin interface.
+
+3. Send the following information to your Open Forms supplier in a secure way:
+
+   * Public and private certificate (obtained in step 2, private certificate is already
+     present if you generated the CSR via the admin)
```

Comment:
Maybe expand certificate signing request?

Reply:
I'm a bit torn about this, because the admin literals use "Signing request" and the CSR abbreviation is used on the change form page for a signing request.